bootstrap.sh

```diff
@@ -5,11 +5,12 @@ git submodule update
 
 GOBIN="$(pwd)/util" GOPATH="$(pwd)/util/third_party:$(pwd)/util" go get -d harvey/cmd/... # should really vendor these bits
 GOBIN="$(pwd)/util" GOPATH="$(pwd)/util/third_party:$(pwd)/util" go install github.com/rminnich/ninep/srv/examples/ufs harvey/cmd/...
- 
-echo Downloading the build tool...
-curl -L http://sevki.co/get-build -o util/build
-chmod +x util/build
- 
+
+if [ $OLD_BUILD != true ]; then
```

Comment:
for now, we need both tools. Also you're still extracting into build, not blaze. So we won't get a complete tree.

This was why I had to remove the new build tool last time: it broke stuff.

Let's take this a bit more slowly. Can you submit JUST a PR that adds all the BUILD files? This is too much to digest. For me, anyway.